code/game/machinery/computer/security.dm

```diff
@@ -754,10 +756,11 @@ What a mess.*/
 								return
 							active1.fields["species"] = t1
 					if("show_photo_front")
-						if(active1.fields["photo_front"])
-							if(istype(active1.fields["photo_front"], /obj/item/photo))
-								var/obj/item/photo/P = active1.fields["photo_front"]
-								P.show(usr)
+						if(active1)
+							var/front_photo = active1.get_front_photo()
+							if(front_photo && istype(front_photo, /obj/item/photo))
```

Comment:
there's no reason to have both of these, null isn't typed as photos

Reply:
Good point!